docs/index.html

```diff
@@ -101,7 +103,7 @@ <h3>Supported data sources</h3>
             <div class="col-xs-2 text-right"><i class="fa fa-2x fa-rss"></i></div>
             <div class="col-xs-10">
               <p>Others</p>
-              <p class="text-justify small">DockerHub, <abbr title="Really Simple Syndication">RSS</abbr></p>
+              <p class="text-justify small">DockerHub, events, GitHub repository stats (<em>stars, watchers, forks</em>),  Google hits, <abbr title="Really Simple Syndication">RSS</abbr>, Twitter</p>
```

Comment:
missing `GitHub` before events?

Reply:
Oops. No, I wanted to remove the whole GitHub events, because they are really GitHub project boards events. Fix coming!